### ISSUE TITLE
can freqai predict labels for higher timeframes in include_timeframes?

### ISSUE THREAD

**Issue by mhgutier** [freqtrade/freqtrade#7902]
## Describe your environment

  * Operating system: mac OS ventura 13.0.1
  * Python Version: 3.8.13
  * CCXT version: NA im using docker
  * Freqtrade Version: freqtrade docker-2022.12.dev-28282554
  
## Your question

Hi,
can freqai predict higher timeframes in include_timeframes?

`df["&-s_close"] = (df["close"].shift(-self.freqai_info["feature_parameters"]["label_period_candles"]))`
`df["&-s_close_15m"] = (df[f"{pair}close_15m"].shift(-self.freqai_info["feature_parameters"]["label_period_candles"]))`
`df["&-s_close_1h"] = (df[f"{pair}close_1h"].shift(-self.freqai_info["feature_parameters"]["label_period_candles"]))`

basically i want to long if all predictions are pointing up, then short when all timeframes are predicted to go down.

**Comment by xmatthias** [comment]
The predictions (and signals) will always be in the base timeframe (the timeframe freqtrade runs on - configured as `timeframe`).

`timeframe` must be the smallest of the timeframes - as you can't reliably merge smaller timeframes to bigger timeframes (without losing all information, basically converting the small timeframe to a (wrong) big timeframe value.

---

You can however have freqAI predict multiple columns by using one of the MultiOutput models (`CatboostClassifierMultiTarget`, `CatboostRegressorMultiTarget`, `LightGBMClassifierMultiTarget`, `LightGBMRegressorMultiTarget`, `XGBoostRegressorMultiTarget`).

**Comment by mhgutier** [comment]
thanks @xmatthias for the quick response.

**Comment by robcaulk** [comment]
Yes, you can absolutely do that. Just ensure you scale your `"label_period_candles"` accordingly (assuming `5m` base tf):

```python
df["&-s_close"] = (df["close"].shift(-self.freqai_info["feature_parameters"]["label_period_candles"]))
df["&-s_close_15m"] = (df[f"{pair}close_15m"].shift(-self.freqai_info["feature_parameters"]["label_period_candles"]*3))
df["&-s_close_1h"] = (df[f"{pair}close_1h"].shift(-self.freqai_info["feature_parameters"]["label_period_candles"]*12))
```

And yes, as matthias says, make sure to use one of the MultiTarget models. But contrary to what matthias recommends, `ReinforcementLearner_multiproc` is not useful here.

**Comment by xmatthias** [comment]
> And yes, as matthias says, make sure to use one of the MultiTarget models. But contrary to what matthias recommends, `ReinforcementLearner_multiproc` is not useful here.

well yes - i copied everything with "multi" without thinking too much

**Comment by mhgutier** [comment]
> 

cool. Thanks Robert and Matthias
